ppafm/GUIWidgets.py

```diff
@@ -413,22 +419,27 @@ def __init__(self, parent=None, title="View Forcefield", width=5, height=4, dpi=
         components = ['Fx', 'Fy', 'Fz', 'E']
 
         hb = QtWidgets.QHBoxLayout(); self.centralLayout.addLayout(hb)
-        hbl = QtWidgets.QHBoxLayout(); hb.addLayout(hbl)
-        hbr = QtWidgets.QHBoxLayout(); hb.addLayout(hbr)
 
+        hbl = QtWidgets.QHBoxLayout(); hb.addLayout(hbl)
         lb = QtWidgets.QLabel('Component:'); lb.setToolTip(tooltips[0]); hbl.addWidget(lb)
         sl = QtWidgets.QComboBox(); self.slComponent = sl; sl.addItems(components)
         sl.setCurrentIndex(sl.findText(components[2])); sl.currentIndexChanged.connect(self.updateView)
         sl.setToolTip(tooltips[0]); hbl.addWidget(sl)
-        hbl.addStretch(1)
```

Comment:
I hope that you know, what you are doing here.

Reply:
This and the following margin change were just to make it look a bit better with the addition of the button for saving the force field.